pymc3/step_methods/smc.py

```diff
@@ -164,26 +165,32 @@ def sample_smc(draws=5000, step=None, progressbar=False, model=None, random_seed
         acc_rate = accepted / proposed
         stage += 1
 
-    trace = _posterior_to_trace(posterior, model)
+    trace = _posterior_to_trace(posterior, model, var_info)
 
     return trace
 
-# FIXME!!!!
-def _initial_population(samples, model, variables):
+
+def _initial_population(chains, model, variables):
```

Comment:
Any reason not using `sample_prior_predictive`?

Reply:
not really, I will change it.